react-frontend/src/components/StyleComponents/nav.js

```diff
@@ -21,3 +93,22 @@ export const NavTitle = styled.p.attrs({
     margin: 0px 5px 0px -5%;
   }
 `;
+
+export const SkipToMainContent = styled.a.attrs({
+  className: 'skipToMainContent',
+  accesskey: '0',
+  href: '#main-content-anchor',
+  'aria-label': 'Skip to main content',
+})`
+  position: absolute;
+  left: -10000px;
```

Comment:
What does this do?

Reply:
This is the suggested method of keeping the 'skip link' invisible unless the element is in focus.   (Mark changed it a bit in his PR, but the idea is the same).  When the element comes into 'focus' the css moves it back onto the page.